module/solutions/helper.js

```diff
@@ -2297,3 +2402,4 @@ function _generateLink(appsPortalBaseUrl, prefix, solutionLink, solutionType) {
 
   return link;
 }
```

Comment:
@VISHNUDAS-tunerlabs  will this not create circular dependency?

Reply:
@ankitpws No